File: CSharp/2175.cs
using System;
using System.Linq;
using System.Globalization;

class URI 
{
    static void Main(string[] args) 
    {   
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        decimal[] n = Console.ReadLine().Split().Select(decimal.Parse).ToArray();

        if(n[0]< n[1] &&n[0]<n[2])
        {Console.WriteLine("Otavio");}
        else if(n[1]< n[0] &&n[1]<n[2])
        {Console.WriteLine("Bruno");}
        else if(n[2]< n[0] &&n[2]<n[1])
        {Console.WriteLine("Ian");}
        else{Console.WriteLine("Empate");}
    }
}

File: Python/2175.py
n=list(map(float,input().split()))
if n[0]<n[1] and n[0]<n[2]:
    print("Otavio")
elif n[1]< n[0] and n[1]<n[2]:
    print("Bruno")
elif n[2]< n[0] and n[2]<n[1]:
    print("Ian")
else:
    print("Empate")